### PR TITLE
Fixes cmp.dm runtime

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

free objective and random are strings not atoms
possibly solves runtime when list of names is passed to cmp instead list of players

## Changelog
:cl:
fix: Fixed runtime in cmp.dm
/:cl:

### CODE REVIEW

File: code/modules/admin/team_panel.dm [hunk 1]
```diff
@@ -147,11 +147,12 @@
...
 
 /datum/team/proc/admin_add_member(mob/user)
-	var/list/minds = list()
-	for(var/mob/M in GLOB.mob_list)
-		if(M.mind)
-			minds |= M.mind
-	var/datum/mind/value = input("Select new member:", "New team member", null) as null|anything in sortNames(minds)
+	var/list/candidates = list()
+	for(var/mob/M in GLOB.player_list)
+		if(M?.mind.special_role)
```
Comment (utilbird) [inline code/modules/admin/team_panel.dm]:
```suggestion
		if(M.mind?.special_role)
```

File: code/modules/admin/team_panel.dm [hunk 2]
```diff
@@ -147,11 +147,12 @@
...
+			continue
+		candidates |= M.mind
```
Comment (utilbird) [inline code/modules/admin/team_panel.dm]: Could probably just use += here, I don't see why there would be multiple instances of the same mob in player list

File: code/modules/antagonists/brother/brother.dm [hunk 3]
```diff
@@ -76,10 +76,10 @@
 /datum/antagonist/brother/admin_add(datum/mind/new_owner,mob/admin)
 	//show list of possible brothers
 	var/list/candidates = list()
-	for(var/mob/living/L in GLOB.alive_mob_list)
+	for(var/mob/living/L in GLOB.player_list)
 		if(!L.mind || L.mind == new_owner || !can_be_owned(L.mind))
 			continue
-		candidates[L.mind.name] = L.mind
+		candidates = L.mind
```
Comment (Archanial) [inline code/modules/antagonists/brother/brother.dm]:
```suggestion
		candidates += L.mind
```

File: code/modules/admin/team_panel.dm (hunk 2 above)
Comment (Archanial) [reply code/modules/admin/team_panel.dm]:
```suggestion
		candidates += M.mind
```

**Review by utilbird — APPROVED** [review]

**Pull request merged**